internal/service/services/services.go

```diff
@@ -285,6 +286,7 @@ func patchService(current openapi.ServiceDto, patch openapi.ServicePatchDto) ope
 		TimeStamp:       patch.TimeStamp,
 		CommitHash:      patch.CommitHash,
 		JiraIssue:       patch.JiraIssue,
+		Description:     patch.Description,
```

Comment:
HTTP PATCH is supposed to have all fields optional, and if they are not present, they are not changed.

With this implementation, if the description is not present in the patch, it is deleted.

You need a method like patchString() here, but for a string pointer.

Reply:
It would probably be good if setting patch.Description to "" leads to deletion of the description in the dto, because having Description as "" in the saved yaml does not really make sense.

Reply:
Updated to use patchStringPtr() -> use original if patch is nil, update if patch is a none empty string, remove if patch is an empty string